### PR TITLE
Adding standard analyzer docs

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
Adding standard analyzer docs

### Issues Resolved
Part of https://github.com/opensearch-project/documentation-website/issues/1483 addressed in this PR.

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@udabhas @varun-lodaya Please provide tech review approval to move this PR forward in the documentation process. Please review this week or provide a peer who can review it. Thank you.

**Review by udabhas — APPROVED** [review]
Looks good to me!

File: _analyzers/standard.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: Standard analyzer
+nav_order: 40
+---
+
+# Standard analyzer
+
+The `standard` analyzer is the default analyzer that is used when no other analyzer is specified. It is designed to provide a basic and efficient approach for generic text processing.
```
Comment (natebower) [inline _analyzers/standard.md:9]:
```suggestion
The `standard` analyzer is the default analyzer used when no other analyzer is specified. It is designed to provide a basic and efficient approach to generic text processing.
```

File: _analyzers/standard.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+This analyzer consists of the following tokenizers and token filters:
+
+- `standard` tokenizer: Removes most punctuation and splits text on spaces and other common delimiters.
```
Comment (natebower) [inline _analyzers/standard.md:13]:
```suggestion
- `standard` tokenizer: Removes most punctuation and splits text at spaces and other common delimiters.
```

File: _analyzers/standard.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+title: Standard analyzer
+nav_order: 40
+---
+
+# Standard analyzer
+
+The `standard` analyzer is the default analyzer that is used when no other analyzer is specified. It is designed to provide a basic and efficient approach for generic text processing.
+
+This analyzer consists of the following tokenizers and token filters:
+
```
Comment (natebower) [inline _analyzers/standard.md:12]: Line 13: I believe we had been using "split text at" (as opposed to "on"), but "on" started appearing in recent docs. Please ensure global consistency.

File: _analyzers/standard.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+- `standard` tokenizer: Removes most punctuation and splits text on spaces and other common delimiters.
+- `lowercase` token filter: Converts all tokens to lowercase, ensuring case-insensitive matching.
+- `stop` token filter: Removes common stopwords such as "the", "is", and "and" from the tokenized output.
```
Comment (natebower) [inline _analyzers/standard.md:15]:
```suggestion
- `stop` token filter: Removes common stopwords, such as "the", "is", and "and", from the tokenized output.
```

File: _analyzers/standard.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+```
+{% include copy-curl.html %}
+
+## Parameters
+
+You can configure a `standard` analyzer with the following parameters.
+
+Parameter | Required/Optional | Data type | Description
+:--- | :--- | :--- | :--- 
+`max_token_length` | Optional | Integer | Sets the maximum length of the produced token. If this length is exceeded, the token is split into multiple tokens at the length configured in the `max_token_length`. Default is `255`.
```
Comment (natebower) [inline _analyzers/standard.md:42]:
```suggestion
`max_token_length` | Optional | Integer | Sets the maximum length of the produced token. If this length is exceeded, the token is split into multiple tokens at the length configured in `max_token_length`. Default is `255`.
```

**Review by natebower — COMMENTED** [review]
@kolchfa-aws @AntonEliatra Editorial review. Thanks!

File: _analyzers/standard.md (hunk 3 above)
Comment (kolchfa-aws) [reply _analyzers/standard.md:12]: Will address this as a separate PR globally.

**Review by kolchfa-aws — APPROVED** [review]

File: _analyzers/standard.md (hunk 2 above)
Comment (kolchfa-aws) [reply _analyzers/standard.md:13]: Will consistently use "on".

**Pull request merged**